electron/src/menu/system.ts

```diff
@@ -241,7 +241,7 @@ const windowTemplate: MenuItemConstructorOptions = {
       label: locale.getText('menuActualSize'),
     },
     {
-      accelerator: 'CmdOrCtrl+Plus',
+      accelerator: 'CmdOrCtrl+=',
```

Comment:
Seems like <kbd>Ctrl</kbd>+<kbd>Plus</kbd> does not actually trigger <kbd>+</kbd> while <kbd>Ctrl</kbd>+<kbd>=</kbd> does.

See https://github.com/electron/electron/issues/1507#issuecomment-118424331.

Reply:
Ok, please add this info (issue link) to the code itself.